minimal-deployment/minimal-deployment-values.yaml

```
@@ -0,0 +1,47 @@
+amalthea:
+  deployCrd: false
```

For a functional deployment from nothing in a fresh cluster this should be true.

```suggestion
  deployCrd: true
```